contrib/facter.ps1

```diff
@@ -127,6 +127,7 @@ if ($buildSource) {
     "--with-regex",
     "--with-log",
     "--with-locale",
+    "--with-chrono",
```

Comment:
And everywhere it was already happening implicitly because Boost's build system new it was required by Boost.Log. That's why CI passed.